CMakeLists.txt

```
@@ -236,6 +248,69 @@ else()
   message(STATUS "PETSc support disabled")
 endif()
 
+# Option: Ginkgo
+if (PRECICE_GinkgoParallelization)
+  find_package(Ginkgo 1.5 REQUIRED)
+  message(STATUS "Found Ginkgo ${GINKGO_PROJECT_VERSION} at ${GINKGO_INSTALL_PREFIX}")
+
+  # We aim for the same configuration as Ginkgo itself and check immediately for device compiler
+  # such that preciceCore inherits all relevant flags directly
+  # 1. Check if Ginkgo was built with CUDA and if CUDA is available on the system
+  if(${GINKGO_BUILD_CUDA})
```

You can directly specify the CMake var in an if

```suggestion
  if(GINKGO_BUILD_CUDA)
```